src/GraphQL.SystemReactive/SubscriptionExecutionStrategy.cs

```diff
@@ -103,9 +103,10 @@ public override async Task<ExecutionResult> ExecuteAsync(ExecutionContext contex
                 }
 
                 return subscription
-                    .Select(value => BuildSubscriptionExecutionNode(node.Parent!, node.GraphType!, node.Field, node.FieldDefinition, node.IndexInParentNode, value!))
-                    .SelectMany(async executionNode =>
+                    .SelectAsync(async (value, token) =>
                     {
+                        //note: token is unused here
```

Comment:
Need to think about this: is it correct for the execution to use the ExecutionContext's cancellation token?  When would that trigger?  Seems like in this case it should cancel when the observable sequence is disconnected, which would be `token` here in my code.  Might need to wrap/duplicate the execution context.  Need to know more about where the execution context's cancellation token came from in order to answer this question.

Reply:
`GraphQLHttpMiddleware.GetCancellationToken`